src/app/modules/useractivitylog/useractivitylog.component.html

```diff
@@ -0,0 +1,93 @@
+<div class="content">
+<md-grid-list cols="8" rowHeight="80px">
+  <md-grid-tile [colspan]="7">
+    <h2 class="h2pos">
+      Candidate Log
+    </h2>
+  </md-grid-tile>
+  <md-grid-tile [colspan]="1">
+   <button md-icon-button [mdMenuTriggerFor]="menu">
+   <md-icon>more_vert</md-icon>
+  </button>
+
+<md-menu #menu="mdMenu">
+  <button md-menu-item>
+    <span>Show all logs</span>
+  </button>
+  <button md-menu-item>
+    <span>abhishek@excellencetechnologies.in</span>
```

Comment:
i will remove repeated things.I added these to check design. no need of separate json file. will replace with ngFor once i get data from api